resources/extras/convert.py

```diff
@@ -0,0 +1,107 @@
+from PIL import Image
+import os
+import json
+from colors import rgb, hex
+print(tuple(hex('FFFFFF').rgb))
+convertpath = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)),".."))
+configpath = convertpath+"\\pxls.conf"
+print(configpath)
+
+configfile = open(configpath,"r+")
+config = str(configfile.read())
+palpos = "Null"
+#Finding pallete variable position
+for i in range(len(config)):
+        if config[i] == "p":
+                if config[i+1] == "a":
+                        if config[i+2] == "l":
+                                if config[i+3] == "e":
+                                        if config[i+4] == "t":
+                                                if config[i+5] == "t":
+                                                        if config[i+6] == "e":
+                                                                if config[i-1] == " ":
+                                                                        palpos = i
+                                                                        break
+                                                                if config[i-1] == "\n":
+                                                                        palpos = i
+print("Palette variable character position:", str(palpos))
+paljson = "{\""
+#Preparing for json readability
+for j in range(len(config)-palpos):
+        if config[j+palpos] == " ":
+                paljson = paljson+"\""+config[j+palpos]
+                posa = j
+                break
+        elif config[j+palpos] == ":":
+                paljson = paljson+"\""+config[j+palpos]
+                posa = j
+                break
+        else:
+                paljson = paljson+config[j+palpos]
+for k in range(len(config)-palpos-posa-1):
+        if config[k+palpos+posa+1] == "\n":
+                break
+        paljson = paljson+config[k+palpos+posa+1]
+paljson = paljson+"}"
+paljson = json.loads(paljson)
+paljson = paljson['palette']
+palette = []
+#Converting RRGGBB to (R,G,B)
+for l in range(len(paljson)):
+        current = paljson[l]
+        palette.append(tuple(hex(current[1]+current[2]+current[3]+current[4]+current[5]+current[6]).rgb))
+print("Palette: ",palette)
+#Getting paths
+image = input("Canvas path: ")
```

Comment:
The script shouldn't require user input while running.
```suggestion
image = sys.argv[1] or 'canvas.png'
```